radar-auth-jersey/README.md

```diff
@@ -0,0 +1,57 @@
+# radar-auth-jersey
```

Comment:
Why can't we package it together with radar-auth? Is it due to the dependency to Jersey?

Reply:
Exactly, because of the Jersey dependency. If that's not a problem, we could also package it together.